packages/core/src/components/ui/ReviewsAndRatings/ReviewsAndRatings.tsx

```diff
@@ -1,6 +1,7 @@
+import { type RatingSummaryProps, useUI } from '@faststore/components'
```

Comment:
```suggestion
import { type RatingSummaryProps, useUI } from '@faststore/ui'
```
we usually import all from faststore/ui package.

Reply:
Done https://github.com/vtex/faststore/pull/2695/commits/ba6c10e7c304041a1376bdac2d112e6d22172e63